static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -71,7 +73,33 @@ define([
         }
     }
 
+
+
+    function shouldAutoplay(){
+
+        function isMobile() {
```

Comment:
you probably want https://github.com/guardian/frontend/blob/master/static/src/javascripts-legacy/projects/common/utils/detect.js#L164-L170